api/docs/http_api_index.md

```diff
@@ -1,13 +1,43 @@
 V1 API Services
 ---------------
 
+The V1 API has been removed. All previous routes can now be found under the
+[Master V3](#Master-V3-HTTP-API) API or the [CA V1](#CA-V1-HTTP-API).
```

Comment:
It seems like these docs eventually end up on our docs.puppetlabs.com site (https://docs.puppetlabs.com/references/3.7.latest/developer/file.http_api_index.html), and I don't know what kinds of markdown is allowed, so this might not be the right thing to do to link to other sections on the same page.